components/micro-gateway-core/src/main/ballerina/Ballerina.toml

```diff
@@ -6,7 +6,7 @@ version= "3.1.0"
 target = "java8"
 
     [[platform.libraries]]
-    path = "../../../target/org.wso2.micro.gateway.core-3.1.0.jar"
+    path = "../../../target/org.wso2.micro.gateway.core-3.1.1-SNAPSHOT.jar"
```

Comment:
We need to update master version number to 3.2.0 or next major version 4.0. Then these values may change. So hold on 😄